trade-finance-manager-api/src/graphql/reducers/mappings/facilities/mapCoverEndDate.js

```diff
@@ -1,20 +1,37 @@
-const moment = require('moment');
+const { fromUnixTime, format } = require('date-fns');
 const { hasValue } = require('../../../../utils/string');
 const { formatYear } = require('../../../../utils/date');
+const api = require('../../../../v1/api');
+const { latestAmendmentCoverEndDateAccepted } = require('../../../helpers/amendment.helpers');
 
-const mapCoverEndDate = (day, month, year) => {
+const mapCoverEndDate = async (day, month, year, facilitySnapshot) => {
   const hasCoverEndDate = (hasValue(day)
     && hasValue(month)
```

Comment:
can we put these conditions in 1 line? it looks like we are declaring new variables this way

Reply:
done